exec/java-exec/src/main/resources/drill-module.conf

```diff
@@ -338,3 +338,127 @@ drill.exec: {
 drill.jdbc: {
   batch_queue_throttling_threshold: 100
 }
+
+drill.exec.options:  {
```

Comment:
Comment to explain what this is? And to remind people how to use it: create a Validator. Create an entry below with the same name as the session/system option, with a default of a compatible type.